gslab_scons/builders/build_tables.py

```diff
@@ -23,16 +24,40 @@ def build_tables(target, source, env):
     # Prelims
     source = misc.make_list_if_string(source)
     target = misc.make_list_if_string(target)
+
+    start_time  = misc.current_time()
     
     # Set up source file (table format)
     source_file = str(source[0])
     misc.check_code_extension(source_file, '.lyx')
 
+    # Set up input string (list of data tables)
+    input_string = ' '.join([str(i) for i in source[1:]])
+
     # Set up target file (filled table)
     target_file = str(target[0])
+    target_dir  = misc.get_directory(target_file)    
+    log_file = target_dir + '/sconscript.log'
     misc.check_code_extension(target_file, '.lyx')
     
-    tablefill(input    = ' '.join([str(i) for i in source[1:]]), 
-              template = source_file, 
-              output   = target_file)
+    # Command call
+    command = """tablefill(input = %s, 
+                 template = %s, 
+                 output   = %s)""" % (input_string, source_file, target_file)
+    output  = tablefill(input    = input_string, 
+                        template = source_file, 
+                        output   = target_file)
+
+    if "error" in str.lower(output): # if tablefill.py returns an error            
```

Comment:
Ah good catch, I was going off the "Error Found" in `tablefill.py` which is not printed. I'll switch it to "traceback" (which we know will be there because `tablefill.py` returns `traceback.formatexc()`